orderer/common/localconfig/config.go

```diff
@@ -294,13 +300,28 @@ func Load() (*TopLevel, error) {
 		return nil, fmt.Errorf("Error reading configuration: %s", err)
 	}
 
-	var uconf TopLevel
-	if err := viperutil.EnhancedExactUnmarshal(config, &uconf); err != nil {
-		return nil, fmt.Errorf("Error unmarshaling config into struct: %s", err)
+	uconf := &TopLevel{}
```

Comment:
The previous var was good; not sure why it had to change. (Doesn't hurt anything...)